lib/tasks/terraform.rake

```diff
@@ -73,17 +72,15 @@ def _purge_remote_state
   FileUtils.rm state_file if File.exist? state_file
 
   if File.exist? state_file
-    warn 'state file should not exist.'
-    exit 1
+    abort('state file should not exist.')
```

Comment:
Can we change this to display the state file that shouldn't exist in the output?